dashboard/Dockerfile

```diff
@@ -0,0 +1,48 @@
+
+FROM node:16-alpine AS pruner
+RUN apk add --no-cache libc6-compat python3 py3-pip make g++
+RUN apk update
+WORKDIR /app
+
+RUN yarn global add turbo
+COPY . .
+RUN turbo prune --scope="@nhost/dashboard" --docker
+
+FROM node:16-alpine AS builder
+RUN apk add --no-cache libc6-compat python3 py3-pip make g++
+RUN apk update
+WORKDIR /app
+
+ENV NEXT_TELEMETRY_DISABLED 1
+ENV NEXT_PUBLIC_NHOST_PLATFORM false
+ENV NEXT_PUBLIC_NHOST_BACKEND_URL http://localhost:1337
```

Comment:
instead of `backendUrl`, we can use `subdomain` https://github.com/nhost/nhost/blob/39de01e3c69f191c62413c9a02fa8d37c49c6579/dashboard/src/utils/nhost.ts#L4

```ts
const client = new NhostClient({
  subdomain: 'localhost'
})
```